ihmc-footstep-planning/src/main/java/us/ihmc/footstepPlanning/SwingPlanningModule.java

```diff
@@ -67,12 +67,13 @@ public void computeSwingWaypoints(HeightMapData heightMapData,
                                      SideDependentList<? extends Pose3DReadOnly> startFootPoses,
                                      SwingPlannerType swingPlannerType)
    {
-      swingTrajectories.clear();
       if (heightMapData == null || heightMapData.isEmpty())
       {
          return;
       }
 
+      swingTrajectories.clear();
```

Comment:
why move this here? In this case, you'll hold onto the old swing trajectories if there's no height map data passed in, which is probably undesirabale behavior. Or is there another reason?

Reply:
Talked in person, could create some weird edge cases, so will move the clear back